### DISCUSSION TITLE
Jetson Orin Agx torch.cuda.is_available() is False

### DISCUSSION THREAD

**Discussion by sm1045** [openai/whisper#1273] [Q&A] [answered]
Good afternoon. I am trying to run whisper into a Jetson Orin Agx. I complete the configuratión of the device according to the requirements suggested for nvidia. When I try to make a translation throws me this error: 

`__RuntimeError: Attempting to deserialize object on a CUDA device but torch.cuda.is_available() is False. If you are running on a CPU-only machine, please use torch.load with map_location=torch.device('cpu') to map your storages to the CPU._`

Can anyone helps me plz? 

p/d: sorry for my bad written English

**Comment by sm1045** [comment]
CUDA compiler seems to be ok. But, I can´t achieve that whisper could use it as a device.

```
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2022 NVIDIA Corporation
Built on Sun_Oct_23_22:16:07_PDT_2022
Cuda compilation tools, release 11.4, V11.4.315
Build cuda_11.4.r11.4/compiler.31964100_0
```

**Reply by sm1045** [reply]
Also, I leave here more information.

`Python version: 3.8.10 (default, Mar 13 2023, 10:26:41)  [GCC 9.4.0] (64-bit runtime)
Python platform: Linux-5.10.104-tegra-aarch64-with-glibc2.29
Is CUDA available: False
CUDA runtime version: 11.4.315
CUDA_MODULE_LOADING set to: N/A
GPU models and configuration: Could not collect
Nvidia driver version: Could not collect
`

**Comment by sm1045** [comment, marked as answer]
Downgrading torch version and adding env var related to CUDA was the solution

**Reply by iamwavecut** [reply]
Please, describe the versions and ENV vars.
